content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -115,20 +112,31 @@ For additional examples, see the automatic trace ID injection projects using [NL
 [1]: https://github.com/DataDog/dd-trace-dotnet/blob/master/samples/AutomaticTraceIdInjection/NLog40Example/NLog.config
 [2]: https://github.com/DataDog/dd-trace-dotnet/blob/master/samples/AutomaticTraceIdInjection/NLog45Example/NLog.config
 [3]: https://github.com/DataDog/dd-trace-dotnet/blob/master/samples/AutomaticTraceIdInjection/NLog46Example/NLog.config
+{{% /tab %}}
+{{% tab "Microsoft.Extensions.Logging" %}}
+If you are using `Microsoft.Extensions.Logging`, no additional setup is required. 
+
```

Comment:
On another note, the ILogger logs injection requires automatic instrumentation. I think we may need to have a separate step for ILogger that ensures that automatic instrumentation is set up. What do you think of changing the getting started section to make the logging library selection the first step and the rest of the content is embedded into each tab (creating some duplication)?

Reply:
Yes, ILogger requires automatic instrumentation (to be honest, I only recently twigged that the other providers _don't_ require it!) 😄 

It's only a small part of duplication, so I think that's ok. It will also make updating for log4Net easier once we switch that to auto-instrumentation

Reply:
Made this change in https://github.com/DataDog/documentation/pull/11623/commits/18cb25cb6ac91aedf4fadc23007a199a88beca5b